pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -668,7 +667,7 @@ func (o *MustGatherOptions) copyFilesFromPod(pod *corev1.Pod) error {
 
 	// get must-gather gather container logs
 	if err := func() error {
-		dest, err := os.OpenFile(path.Join(destDir, "/gather.logs"), os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0755)
+		dest, err := os.OpenFile(path.Join(destDir, "/gather.logs"), os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0o755)
```

Comment:
Is there any particular reason of this change? or a simple refactoring?

Reply:
I have gofumpt set up to run every time I save a file, that's the source of the change. Apparently there's a rule there to prefer formatting octal numbers with `0o755` instead of `0755`. I separated the formatting changes on their own commit: https://github.com/openshift/oc/pull/1790/commits/1c6c3169f1ff86282d9df86574efc2f18a4e0321
Can remove the commit if desired, but I saw no harm. gofumpt is supposed to be a bit stricter than gofmt.

Reply:
Yes, I agree no harm and we can make this change. Asked just for curiosity.